erizo/src/erizo/thread/IOWorker.h

```diff
@@ -8,12 +8,18 @@
 #include <thread>  // NOLINT
 #include <vector>
 
+#include "./logger.h"
+#include "lib/LibNiceInterface.h"
```

Comment:
We don't actually include glib directly anywhere in the licode code, so it would require some tweaking in the build process. I could include libnice (agent, etc) directly but I prefer this.